server/frontend/src/api.js

```diff
@@ -64,3 +64,43 @@ export const listPools = async (params) =>
 
 export const listTasks = async (params) =>
   (await mainAxios.get("/taskmanager/rest/tasks/", { params })).data;
+
+export const covManagerDiffStats = async ({ path, params, cb }) =>
+  await mainAxios
```

Comment:
You should only call await inside of an async function if you need the result.  Otherwise return the promise.